src/Aspire.Hosting/Dashboard/proto/resource_service.proto

```diff
@@ -210,6 +210,7 @@ message Resource {
 
     // The aggregate health state of the resource. Generally reflects data from health_reports, but may differ.
     optional HealthStatus health_status = 16;
```

Comment:
Was there offline discussion about adding this back? I'm not against it, just want to make sure that was an agreed decision.

Reply:
No we want to remove it

Reply:
I know I said before I didn't want it, but sending the calculated value does reduce duplication. It lets the dashboard be more dumb and accept whatever status the service decides 😄 

I haven't got a hard opinion either way, but the current code looks fine to me.

Reply:
@davidfowl I had re-added it to the proto to do what james said - avoid re-calculating the status in the dashboard and avoid duplication

Reply:
Remove it, let the dashboard recalculate it.

Reply:
Re-removed :) is everything else ok?